tensorflow/workspace2.bzl

```
@@ -204,9 +204,13 @@ def _tf_repositories():
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
         patch_file = [
             "//third_party/mkl_dnn:onednn_acl_threadcap.patch",
+            "//third_party/mkl_dnn:onednn_acl_remove_winograd.patch",
             "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch",
             "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch",
             "//third_party/mkl_dnn:onednn_acl_threadpool_scheduler.patch",
+            "//third_party/mkl_dnn:onednn_reorder_padded.patch",
```

Should the file name has `acl`?
```suggestion
            "//third_party/mkl_dnn:onednn_acl_reorder_padded.patch",
```

---

Yeah that's better, fixed.